src/renderers/shared/shared/__tests__/ReactEmptyComponent-test.js

```diff
@@ -246,14 +246,20 @@ describe('ReactEmptyComponent', () => {
     expect(assertions).toBe(3);
   });
 
-  it('throws when rendering null at the top level', () => {
-    // TODO: This should actually work since `null` is a valid ReactNode
+  it('can render null at the top level', () => {
     var div = document.createElement('div');
-    expect(function() {
+
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
I was going to start working on that—but happy to have @acdlite fix it. I didn't do it initially because I thought we're going to introduce a flag to disable *all* Fiber features including frags, and do it in a single pass.